features/addressing/managing_countries/editing_country_with_provinces.feature

```diff
@@ -0,0 +1,26 @@
+@addressing
+Feature: Editing country with provinces
+    In order to add or remove provinces in existing countries
+    As an Administrator
+    I want to be able to edit a country and its provinces
+
+    Background:
+        Given I am logged in as an administrator
+        And the store has country "United Kingdom"
+
+    @ui @javascript
+    Scenario: Adding province to existing country
+        Given I want to edit this country
```

Comment:
Because regarding the scope of the scenario we do not know which country it is. Even though there is only one defined in the shop by that I am forcing the reader to go back and read the background. 
IMHO using the country's name is a whole lot more readable and we are really taking care of it in the new Behats. :)